database/sqlite/sqlite_metadata.c

```diff
@@ -685,6 +685,16 @@ static void check_dimension_metadata(struct metadata_wc *wc)
         error_report("Failed to finalize the prepared statement when reading dimensions");
 }
 
+static void cleanup_health_log(void)
+{
+    RRDHOST *host;
+    dfe_start_reentrant(rrdhost_root_index, host) {
+        if (rrdhost_flag_check(host, RRDHOST_FLAG_ARCHIVED) || !rrdhost_flag_check(host, RRDHOST_FLAG_METADATA_UPDATE))
```

Comment:
Why check for RRDHOST_FLAG_METADATA_UPDATE ? This will probably skip cleanup

Reply:
You are right, removed.